rllib/policy/eager_tf_policy.py

```diff
@@ -724,52 +661,130 @@ def set_state(self, state):
             # Then the Policy's (NN) weights.
             super().set_state(state)
 
+        @override(Policy)
+        def export_checkpoint(self, export_dir):
+            raise NotImplementedError  # TODO: implement this
+
+        @override(Policy)
+        def export_model(self, export_dir):
+            raise NotImplementedError  # TODO: implement this
+
         def variables(self):
             """Return the list of all savable variables for this policy."""
             if isinstance(self.model, tf.keras.Model):
                 return self.model.variables
             else:
                 return self.model.variables()
 
-        @override(Policy)
-        def is_recurrent(self):
-            return self._is_recurrent
+        def loss_initialized(self):
+            return self._loss_initialized
 
-        @override(Policy)
-        def num_state_tensors(self):
-            return len(self._state_inputs)
+        @with_lock
+        def _compute_actions_helper(self, input_dict, state_batches, episodes,
+                                    explore, timestep):
+            # Increase the tracing counter to make sure we don't re-trace too
+            # often.
+            self.re_trace_counter += 1
```

Comment:
can we add a comment here saying:

this python statement is only executed during tracing.
we use self._traced_compute_actions_helper for actual action computation.
therefore, self._re_trace_counter should exactly equal to the number of times _compute_actions_helper is traced.

something like that. do I understand this correctly?

Reply:
Great idea! Will add this comment.

Reply:
done